documentation/source/_static/css/custom.css

```diff
@@ -286,3 +286,10 @@ code.literal {
     border-right: 1px solid var(--color-sidebar-background-border);
     background: #3d3d3c !important;
 } */
+
+/* Figure caption text */
+div.figure p.caption span.caption-text,
+figcaption span.caption-text {
+    font-style: italic;
+    font-size: 0.78rem;
+}
```

Comment:
Note: This CSS code was modeled after Sphinx's `basic.css` file, which provides a template of properties you can modify:

```css
/* -- figures --------------------------------------------------------------- */

div.figure, figure {
    margin: 0.5em;
    padding: 0.5em;
}

div.figure p.caption, figcaption {
    padding: 0.3em;
}

div.figure p.caption span.caption-number,
figcaption span.caption-number {
    font-style: italic;
}

div.figure p.caption span.caption-text,
figcaption span.caption-text {
}
```

I used the last one. :slightly_smiling_face: